streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -132,7 +134,7 @@ public Builder(final String storeName, final Serde<K> keySerde, final Serde<V> v
 
         @Override
         public Map<String, String> logConfig() {
-            return Collections.emptyMap();
+            return Collections.unmodifiableMap(logConfig);
```

Comment:
According to the inherited javadoc (`If {@code loggingEnabled} returns false, this function will always return an empty map`), this will have to be something more like:
```suggestion
            return loggingEnabled() ? Collections.unmodifiableMap(logConfig) : Collections.emptyMap();
```